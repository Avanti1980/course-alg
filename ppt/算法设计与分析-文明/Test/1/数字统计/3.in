5
3934892699
3084855443061
39726443
38601519
296264556654412