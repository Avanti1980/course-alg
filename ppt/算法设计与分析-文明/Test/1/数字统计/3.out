7291
15907
10946
4126
87